tests/Integration/Behaviour/Features/Scenario/Product/bulk_update_product_status.feature

```diff
@@ -48,7 +48,7 @@ Feature: Bulk update product status from BO (Back Office)
     And product "product3" should be disabled
 
   Scenario: I update product statuses
-    When I bulk change status to be enabled for following products:
+    When I bulk enable following products in default shop:
```

Comment:
```suggestion
    When I bulk enable following products:
```
I don't think we need to specify for default shop, we didn't add it on all the other new end point so we can follow this convention. The new wording is better though!